Source/WebKit/NetworkProcess/cocoa/NetworkSessionCocoa.mm

```diff
@@ -2362,7 +2362,9 @@ void completeWithBlockedError()
         else
             zeroBytes(identifier);
 
-        auto nwProxyConfig = adoptNS(createProxyConfig(config.first.span().data(), config.first.size(), identifier));
+        // This is correct but static analysis does not seem to recognize that `createProxyConfig` returns
+        // a +1 value.
+        SUPPRESS_RETAINPTR_CTOR_ADOPT auto nwProxyConfig = adoptNS(createProxyConfig(config.first.span().data(), config.first.size(), identifier));
```

Comment:
We should annotate the declaration of `nw_proxy_config_create_with_agent_data` with `NW_RETURNS_RETAINED` or `OS_OBJECT_RETURNS_RETAINED`.